styles/base.css

```diff
@@ -42,40 +54,106 @@ a {
     transition: 250ms;
 }
 
-a:hover, a:focus {
+a:hover,
+a:focus {
     color: var(--green);
 }
 
 ul.no-bullets {
+    padding-left: 0;
+}
+
+ul.no-bullets li {
```

Comment:
Could you use this Bootstrap unstyled list instead: https://getbootstrap.com/docs/5.2/content/typography/#unstyled

Reply:
Save this for a future refactor.